saml-core/src/test/java/org/keycloak/saml/common/util/StringUtilTest.java

```diff
@@ -1,47 +0,0 @@
-/*
```

Comment:
This test has been removed since it only tests removed method `getSystemPropertyAsString`